ts/Iterable.ts

```diff
@@ -167,10 +167,10 @@ class Iterable {
   * Set the user of the SDK by email address
   * @param {string | undefined} email the email address of the user
   */
-  static setEmail(email: string | undefined) {
+  static setEmail(email: string | undefined, authToken?: string | undefined) {
```

Comment:
I'm assuming, extra argument in RN is optional and can be passed just one parameter as well. This is cool that theres no need of extra methods.

Reply:
Yep, you can see our other API functions, same thing going on there.